var/spack/repos/builtin/packages/py-scikit-optimize/package.py

```diff
@@ -21,17 +21,24 @@ class PyScikitOptimize(PythonPackage):
     maintainers("liuyangzhuan")
 
     version("master", branch="master")
+    version("0.9.0", sha256="77d8c9e64947fc9f5cc05bbc6aed7b8a9907871ae26fe11997fd67be90f26008")
     version("0.5.2", sha256="1d7657a4b8ef9aa6d81e49b369c677c584e83269f11710557741d3b3f8fa0a75")
 
     variant("plots", default=True, description="Build with plot support from py-matplotlib")
-    variant("gptune", default=False, description="Build with patches for GPTune")
+    variant("gptune", default=False, description="Build with patches for GPTune", when="@0.5.2")
 
     depends_on("py-setuptools", type="build")
     depends_on("py-numpy", type=("build", "run"))
-    depends_on("py-scipy@0.14.0:", type=("build", "run"))
+    depends_on("py-numpy@1.13.3:", when="@0.9:", type=("build", "run"))
+    depends_on("py-scipy@0.14:", type=("build", "run"))
+    depends_on("py-scipy@0.19.1:", when="@0.9:", type=("build", "run"))
     depends_on("py-scikit-learn@0.19.1:", type=("build", "run"))
-    depends_on("py-pyyaml", when="+gptune", type=("build", "run"))
+    depends_on("py-scikit-learn@0.20:", when="@0.9:", type=("build", "run"))
+    depends_on("py-pyyaml", when="@0.5.2+gptune", type=("build", "run"))
```

Comment:
```suggestion
```

@adamjstewart what's the point of this line? It's not a dependency of scikit-optimize, nor is the `space.patch` patch importing it. It should just be dropped.